common/txmgr/broadcaster.go

```diff
@@ -694,37 +705,33 @@ func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) tryA
 			attempt.TxFee, txError.Error(), eb.feeConfig.FeePriceDefault())
 	}
 
-	replacementAttempt, bumpedFee, bumpedFeeLimit, retryable, err := eb.NewBumpTxAttempt(ctx, etx, attempt, nil, lgr)
+	bumpedAttempt, bumpedFee, bumpedFeeLimit, retryable, err := eb.NewBumpTxAttempt(ctx, etx, attempt, nil, lgr)
 	if err != nil {
-		return fmt.Errorf("tryAgainBumpFee failed: %w", err), retryable
+		return bumpedAttempt, fmt.Errorf("replaceAttemptWithBumpedGas failed: %w", err), retryable
 	}
 
-	return eb.saveTryAgainAttempt(ctx, lgr, etx, attempt, replacementAttempt, initialBroadcastAt, bumpedFee, bumpedFeeLimit, retry)
+	err = eb.saveReplacementAttempt(ctx, lgr, attempt, &bumpedAttempt, bumpedFee, bumpedFeeLimit)
```

Comment:
I think in the previous behavior if an error occurred during `SaveReplacementInProgressAttempt` we would explicitly return is a retryable = true. Since we aren't checking the `err` here, I think we could be return retryable = false here.

Reply:
Good catch, I thought about it but I think you are right we should keep it false even for error in estimator/db since it can be transient and for backward compatibility

Reply:
Actually I think always return true only applies for insufficient fund error, for gas bump and invalidRange we return retryable value from attempt creation upon error occur